pyhive/tests/sqlalchemy_test_case.py

```diff
@@ -36,15 +39,15 @@ def wrapped_fn(self, *args, **kwargs):
 class SqlAlchemyTestCase(with_metaclass(abc.ABCMeta, object)):
     @with_engine_connection
     def test_basic_query(self, engine, connection):
-        rows = connection.execute('SELECT * FROM one_row').fetchall()
+        rows = connection.execute(text('SELECT * FROM one_row')).fetchall()
         self.assertEqual(len(rows), 1)
         self.assertEqual(rows[0].number_of_rows, 1)  # number_of_rows is the column name
         self.assertEqual(len(rows[0]), 1)
 
     @with_engine_connection
     def test_one_row_complex_null(self, engine, connection):
-        one_row_complex_null = Table('one_row_complex_null', MetaData(bind=engine), autoload=True)
-        rows = one_row_complex_null.select().execute().fetchall()
+        one_row_complex_null = Table('one_row_complex_null', MetaData(), autoload_with=engine)
```

Comment:
https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#implicit-and-connectionless-execution-bound-metadata-removed